### PR TITLE
Release v3.1.0

### PR DESCRIPTION
**Pull request by szepeviktor**
Let's run CI for the release!

### CODE REVIEW

**Review by IvanGrginovInf — APPROVED** [review]

**Review by iruzevic — CHANGES_REQUESTED** [review]
@dingo-d Is this ready to be merged to develop?

**Comment by dingo-d** [comment]
@iruzevic I wouldn't merge it just yet. Let's up the coverage a bit, fix all the potential bugs and issues first, and then we can merge to develop.

**Comment by szepeviktor** [comment]
Please put `.phpunit.result.cache` into gitignore and remove it from git.

**Comment by szepeviktor** [comment]
Change `CliInterface` return typehint to `self` in `src/Cli/AbstractCli.php`

File: src/Blocks/BlockVariationCli.php [hunk 1]
```diff
@@ -119,16 +119,11 @@ public function __invoke(array $args, array $assocArgs) // phpcs:ignore
...
-
-				$class = $this->renameNamespace($assocArgs, $class);
-
-				$class = $this->renameTextDomainFrontendLibs($assocArgs, $class);
-
-				$class = $this->renameUseFrontendLibs($assocArgs, $class);
-
-				// Output final class to new file/folder and finish.
-				$this->outputWrite($path, $file, $class, ['skip_existing' => true]);
+				$class->renameProjectName($assocArgs)
```
Comment (szepeviktor) [inline src/Blocks/BlockVariationCli.php:122]: Hello! 👋 
I'm the method `renameProjectName`.
I want to be called `changeProjectName` but my parents resist.

Comment (dingo-d) [reply src/Blocks/BlockVariationCli.php:122]: Parents don't really have issues with the naming or changing them tbh 😂 

Why would you rename this method? I mean, it's doing the renaming after all, no?

Comment (szepeviktor) [reply src/Blocks/BlockVariationCli.php:122]: Changing/Replacing placeholders with actual values are called "change" or "replace".

**Review by IvanGrginovInf — APPROVED** [review]

File: src/Blocks/BlockVariationCli.php (hunk 1 above)
Comment (dingo-d) [reply src/Blocks/BlockVariationCli.php:122]: Or renaming from the default, placeholder, name to something other 😉

**Comment by dingo-d** [comment]
There is a problem with using `setup_theme` CLI in this branch. It copies the files that do not exist, so that will have to be addressed.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -58,12 +57,11 @@ jobs:
...
          coverage: none
-         tools: cs2pr
 
      # Install dependencies and handle caching in one go.
      # @link https://github.com/marketplace/actions/install-composer-dependencies
      - name: Install Composer dependencies
        uses: "ramsey/composer-install@v1"
 
      - name: Check code consistency using PHPStan
-       run: composer analyze | cs2pr
```
Comment (szepeviktor) [inline .github/workflows/ci.yml]: @dingo-d PHPStan includes automatic output for GHA

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -29,15 +29,14 @@ jobs:
...
           coverage: none
-          tools: cs2pr
 
       # Install dependencies and handle caching in one go.
       # @link https://github.com/marketplace/actions/install-composer-dependencies
       - name: Install Composer dependencies
         uses: "ramsey/composer-install@v1"
 
       - name: Check coding standards using PHPCS
-        run: composer standards:check -- --runtime-set testVersion ${{ matrix.php }}- | cs2pr
```
Comment (szepeviktor) [inline .github/workflows/ci.yml]:
```yaml
run: composer standards:check -- -q --runtime-set testVersion ${{ matrix.php }}- --report=checkstyle | cs2pr
```

Comment (dingo-d) [reply .github/workflows/ci.yml]: I tried running act with this or with XML but still I get an error. I'll leave it removed for now.

Comment (szepeviktor) [reply .github/workflows/ci.yml]: Maybe `-q` was missing.

**Comment by dingo-d** [comment]
@iruzevic there seems to be one phpstan error that was introduced 🙈

**Comment by iruzevic** [comment]
We will open a PR when we are ready to merge

**Pull request closed without merging**